Ptt/Flows/BoardFlow/Controllers/SingleArticleViewController.swift

```diff
@@ -0,0 +1,359 @@
+//
+//  SingleArticleViewController.swift
+//  Ptt
+//
+//  Created by denkeni on 2020/1/9.
+//  Copyright © 2020 Ptt. All rights reserved.
+//
+
+import UIKit
+import SafariServices
+import AsyncDisplayKit
+
+final class SingleArticleViewController: ASDKViewController<ASDisplayNode>, FullscreenSwipeable, ArticleView {
+
+    private let boardNode = BoardNode()
+    private var tableNode : ASTableNode {
```

Comment:
I don't have strong opinions so up to you.  
Can we follow [this doc](https://realm.github.io/SwiftLint/colon.html)?

For variable declaration, it is `Name : Type`.  
For dictionary, it is `[.font: UIFont.....]`.  
For function, it is `init(article: APIModel.FullArticle)`  
please stay consistency

Reply:
I roughly follow UIKit style for now. I expect to bring the focus on feature development, and enforce Swift style guide later, e.g. after the 1st version being live on the App Store.

Automation tools would be preferred for consistency, e.g. [SwiftFormat](https://github.com/nicklockwood/SwiftFormat) or [SwiftLint](https://github.com/realm/SwiftLint), but I haven't done extensive research yet. If you're interested in implementing this topic, let's open an issue for further discussion.

Reply:
ya, I can integrate it